Connectors/Revit/Speckle.Connectors.RevitShared/Operations/Receive/RevitHostObjectBuilder.cs

```diff
@@ -56,31 +62,37 @@ CancellationToken cancellationToken
 
       var conversionResults = BakeObjects(objectsToConvert);
 
-      _transactionManager.CommitTransaction();
-      transactionGroup.Assimilate();
-
+      using (var _ = SpeckleActivityFactory.Start("Commit"))
```

Comment:
"Commit transaction" would make more sense (to avoid confusion with the former naming of speckle `Version`s)